### DISCUSSION TITLE
re-dowload a blog

### DISCUSSION THREAD

**Discussion by bloke90** [TumblThreeApp/TumblThree#326] [Q&A (Question & Answer)]
I was trying to find here but failed: how can I "urge" a re-download of a blog I already downloaded before? "urge rescan" doesn't help. 
The blogname seems to be saved somewhere, where to reset this?

**Comment by thomas694** [comment]
Your initial situation is not totally clear. "Force rescan" is rescanning/downloading the meta data and checks if something has been missed (due to timeouts etc.) before, but doesn't unnecessarily download again the files.
If you mean, you "lost" your blog folder, you have to get rid of the entries in blog_files.tumblr, either by editing it manually or maybe better: 
- remove the blog from the list
- delete/remove the two blog files from the Archive folder
- add the blog again to the list and download again

**Comment by bloke90** [comment]
thanks!
